dallinger/command_line.py

```diff
@@ -170,6 +170,42 @@ def setup_experiment(debug=True, verbose=False, app=None, exp_config=None):
     cwd = os.getcwd()
     os.chdir(dst)
 
+    # Freeze the Python requirements
+    log('Freezing Python requirements...')
+    blacklist = (
+        # Dev dependencies
+        'alabaster',
+        'coverage',
+        'coverage_pth',
+        'codecov',
+        'flake8',
+        'mock',
+        'pyenchant',
+        'pytest',
+        'Sphinx',
+        'sphinxcontrib-spelling',
+        'tox',
+        # Data analysis tools
+        'datashape',
+        'networkx',
+        'numpy',
+        'odo',
+        'pandas',
+        'tablib',
+        # Obsolete
+        'psiturk-dallinger',
+    )
+    with open(os.path.join(dst, 'requirements.txt'), 'w') as file:
+        for line in subprocess.check_output(['pip', 'freeze']).splitlines():
```

Comment:
It might be nicer to do

    for line in pip.operations.freeze.freeze(skip=blacklist):

here, to avoid a subprocess and post-processing the output.